controllers/lmes/driver/driver.go

```diff
@@ -283,6 +283,29 @@ func (dc *driverComm) notifyShutdownWait() {
 	dc.connection <- 1
 }
 
+type PodLogAndPipeWriter struct {
+	pw *io.PipeWriter
+}
+
+func (f PodLogAndPipeWriter) Write(p []byte) (n int, err error) {
+	// format lm-eval output to stdout and the pipewriter
+	line := string(p)
+	res, _ := fmt.Print(line)
```

Comment:
since this writer prints out the stderr, maybe line 134 should be commented out. or the stderr would be printed again when the lm-eval job finishes.